tests/everflow/test_everflow_ipv6.py

```diff
@@ -99,11 +100,13 @@ def test_src_ipv6_mirroring(self, setup_info, setup_mirror_session, ptfadapter,
                                            ptfadapter,
                                            everflow_dut,
                                            test_packet, everflow_direction, src_port=EverflowIPv6Tests.rx_port_ptf_id,
-                                           dest_ports=EverflowIPv6Tests.tx_port_ids)
+                                           dest_ports=EverflowIPv6Tests.tx_port_ids,
+                                           skip_traffic_test=skip_traffic_test)
```

Comment:
If we need to explicitly specify fixture `skip_traffic_test` here, then this fixture does not need to be autouse. Please always try to avoid using `autouse` fixture unless there is a clear reason.

Reply:
Agree, fixed in new commit